.github/workflows/node.js.yml

```
@@ -5,26 +5,73 @@ name: Node.js CI
+          PLAYWRIGHT_START_SERVER: "true"
+          PLAYWRIGHT_PORT: "5173"
+          PLAYWRIGHT_HOST: "localhost"
+      - name: Upload Playwright Report
+        uses: actions/upload-artifact@v4
+        if: always()
+        with:
+          name: playwright-report
+          path: tools/snapshot-tests/playwright-report/
+          retention-days: 30
+      - name: Deploy to GitHub Pages
+        uses: peaceiris/actions-gh-pages@v3
```

actionlint (1.7.7)

the runner of "peaceiris/actions-gh-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)